src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue30818.cs

```diff
@@ -0,0 +1,39 @@
+#if IOS
```

Comment:
I realize this is just iOS right now, but, I want to get this merged for SR10 and we don't currently need to generate the screen shots for the other platforms in order to merge this.

I'll do a followup PR to get the rest of the platforms baselined.